### ISSUE TITLE
Add "Scroll to see reports" button 

### ISSUE THREAD

**Issue by kosamari** [kosamari/who-pays-tech-speakers#30]
The form is gelling longer and I'm afraid visitors won't notice there is a repots to read at the bottom.
Adding something like ....

```
Share your story in the form below or  [check submitted reports](link to scroll down)
```

would be nice.
I'm also open to any layout change on the site if visitors can look at reports in 1st page view.

![who_pays_conference_speakers_ 2](https://cloud.githubusercontent.com/assets/4581495/10837173/bcd63126-7e8c-11e5-85c3-022fe6570d83.png)

**Comment by droidenator** [comment]
What if we made it so "Share Your Experience" form was collapsed on page load? Depending on the user's browser viewport size, that would bring the reports up higher but I'm not sure if it'll get the reports up where user's will see them on the initial page load.

**Comment by kosamari** [comment]
yey thanks @droidenator :)
merged in #34 (sorry for delay in getting back !)
